admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MasterdataSearchHelperTest.java

```diff
@@ -270,4 +285,291 @@ public void searchStartswithWildcardMasterdata04() {
 		SearchDto searchDto = new SearchDto(Arrays.asList(wildCardFilter7), Arrays.asList(sort), page, "eng");
 		searchHelper.searchMasterdata(RegistrationCenter.class, searchDto, null);	
 	}
+
+	@Test
+	public void searchMasterdataWithoutLangCode_withValidFilterAndSort_returnSearchDtoWithoutLangCode() {
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(filter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
```

Comment:
assertion is missing, please check for all the test cases.

Reply:
Added assertions for missed test cases